src/scaladoc/scala/tools/nsc/doc/html/HtmlPage.scala

```diff
@@ -145,8 +145,12 @@ abstract class HtmlPage extends Page { thisPage =>
         <span class="extmbr" name={ mbr.qualifiedName }>{ inlineToHtml(text) }</span>
     case Tooltip(tooltip) =>
       <span class="extype" name={ tooltip }>{ inlineToHtml(text) }</span>
-    case LinkToExternal(name, url) =>
-      <a href={ url } class="extype" target="_top">{ inlineToHtml(text) }</a>
+    case LinkToExternalTpl(name, baseUrl, dtpl: TemplateEntity) =>
```

Comment:
This is indeed weird. As far as I understand this template is always a `NoDocTemplate` so I'll try to make that explicit. This way the pattern matching should be exhaustive.